tests/std/tests/P0122R7_span/test.cpp

```diff
@@ -761,13 +787,10 @@ constexpr bool test() {
 

```

Comment:
Please verify that this test compiles; it appears that the `static_assert`s above, for span converting construction, need to be changed.